### PR TITLE
Added Commonly asked questions

### PR DESCRIPTION
**Pull request by ihavezerohealth**
Added "Commonly asked questions" section and added a couple of questions that I have seen in the Dank Grinder Discord.
If there are any errors, please tell me, thanks!
It'll help us all out so we don't have to nek snapp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -303,6 +303,16 @@ shifts:
     base: 32400
 ```
 
+### Commonly asked questions
```
Comment (dankgrinder) [inline README.md:306]:
```suggestion
## Commonly asked questions
```

File: README.md [hunk 2]
```diff
@@ -303,6 +303,16 @@ shifts:
...
+## How do I disable certain features?
```
Comment (dankgrinder) [inline README.md:307]:
```suggestion
### How do I disable certain features?
```

File: README.md [hunk 3]
```diff
@@ -303,6 +303,16 @@ shifts:
...
+You can remove a specific feature by removing it from the config, setting its "enable" value to false, or by setting the cooldown at the bottom of the config to a very high number.
+
+## How do I disable global events?
```
Comment (dankgrinder) [inline README.md:310]:
```suggestion
### How do I disable global events?
```

File: README.md [hunk 4]
```diff
@@ -303,6 +303,16 @@ shifts:
...
+This is not possible as of now.
+
+## Can I run it on other architectures?
```
Comment (dankgrinder) [inline README.md:313]:
```suggestion
### Can I run it on other architectures?
```

File: README.md [hunk 5]
```diff
@@ -303,6 +303,16 @@ shifts:
     base: 32400
 ```
 
+### Commonly asked questions
+## How do I disable certain features?
+You can remove a specific feature by removing it from the config, setting its "enable" value to false, or by setting the cooldown at the bottom of the config to a very high number.
```
Comment (dankgrinder) [inline README.md:308]:
```suggestion
You can remove a specific feature by removing it from the config, setting its `enable` value to `false`.
```

**Review by dankgrinder — CHANGES_REQUESTED** [review]
Looks good. I have some minor reservations which I have addressed in my review. If you want to, adding the frequently asked questions which are currently only on Discord would also be appreciated!

**Comment by dankgrinder** [comment]
Still waiting for some requested changes. Do you want to approve those or cancel the PR?

**Comment by ihavezerohealth** [comment]
how do I do that haha

**Comment by Tripp-collins** [comment]
i go banned on the bot i hate this

**Comment by Cherry-Picked-0001** [comment]
Why doesn't this work for bots? I mean shouldn't it be able to send messages if the bot's token is entered. I turned off all the Dank Memer commands and added custom commands but it doesn't work. Does this not work for bots?

**Comment by pythonmcpi** [comment]
> Why doesn't this work for bots? I mean shouldn't it be able to send messages if the bot's token is entered. I turned off all the Dank Memer commands and added custom commands but it doesn't work. Does this not work for bots?

Dankgrinder was made to be used with a user token. The login process for user and bot tokens is slightly different, so you would have to modify the source code (and possibly a lot of it depending on how it is written) to get it working with a bot token. It would be easier to write your own bot in discord.py or discord.js (or other languages).

**Comment by dankgrinder** [comment]
Closing due to inactivity

**Pull request closed without merging**